modin/pandas/__init__.py

```diff
@@ -46,6 +47,11 @@ def get_npartitions():
 try:
     if threading.current_thread().name == "MainThread":
         import ray
-        ray.init()
+        if os.environ.get("MODIN_EXECUTION_FRAMEWORK") == "ray" and \
+                os.environ.get("MODIN_RAY_REDIS_ADDRESS"):
+            redis_address = os.environ.get("MODIN_RAY_REDIS_ADDRESS")
+            ray.init(redis_address=redis_address)
```

Comment:
I suspect it will simplify users' lives to just call `ray.init(redis_addres=...)` but I could be wrong about that.

Reply:
@devin-petersohn thoughts? This was what I was doing in an older version of the PR.